src/components/App.js

```diff
@@ -15,7 +15,6 @@ class App extends Component {
 
   handleClick = (simbolOperation) => {
     const result = Calculate(this.state, simbolOperation);
-    console.log(result);
     this.setState(result);
   }
 
```

Comment:
`<Display value={this.state.next || this.state.total || '0'} />`

Note, we want to keep our components as dry as possible